frontend/src/app/component/ingredients-component/ingredients-component.ts

```diff
@@ -1,11 +1,34 @@
 import { Component } from '@angular/core';
+import { TableModule } from 'primeng/table';
+import {Ingredient, IngredientsService} from '../../services/ingredients-services/ingredients-service';
+import {DatePipe, DecimalPipe} from '@angular/common';
+import {MenuItem} from 'primeng/api';
+import { RouterModule } from '@angular/router';
 
 @Component({
   selector: 'app-ingredients-component',
+  standalone: true,
   templateUrl: './ingredients-component.html',
-  styleUrl: './ingredients-component.css'
+  imports: [TableModule, DatePipe, DecimalPipe, RouterModule],
+  styleUrls: ['./ingredients-component.css']
 })
 
 export class IngredientsComponent {
+  items: MenuItem[] = [];
+  ingredients: Ingredient[] = [];
 
+  constructor(private ingredientsService: IngredientsService) {}
+
+  ngOnInit() {
+    this.ingredientsService.getIngredients().subscribe(data => {
+      this.ingredients = data;
+    });
```

Comment:
This works, but in the long run, it may cause problems due to memory leaks, also since it's an observable which receives one value and then closes we can do the following thing which handles this correctly:
```suggestion
    this.ingredients = await firstValueFrom(this.ingredientsService.getIngredients());
```
you will also have to make the method async in this case